test/controllers/ClientControllerSpec.scala

```diff
@@ -29,10 +30,14 @@ class ClientControllerSpec
 
       val time       = Clock.systemUTC().instant()
       val clientName = s"Automated Test client $time"
-      val request = FakeRequest(
-        POST,
-        s"/clients?name=$clientName&email=some.email@server.invalid"
-      ).withHeaders("Content-type" -> "application/json")
+      val jsonString =
```

Comment:
Parempi rakentaa testidata DTO:n avulla, joka serialisoidaan sitten jsoniksi kutsuun. Näin ollen meillä säilyy tyypitys mukana, jos tuo endpointin vastaanottama body muuttaisi muotoaan. Tälläinen dynaaminen stringi kun ei compiler apua saa